spark/src/test/scala/com/gu/tableversions/spark/SparkHiveSuite.scala

```diff
@@ -77,6 +77,8 @@ trait SparkHiveSuite extends BeforeAndAfterAll with BeforeAndAfterEach with Lazy
       .set("spark.driver.host", "127.0.0.1")
       .set("spark.sql.orc.impl", "native")
       .set("spark.sql.shuffle.partitions", "8") // For speeding up tests. The default is 200, see https://spark.apache.org/docs/latest/sql-programming-guide.html#other-configuration-options
+      .set("fs.versioned.impl", "com.gu.tableversions.spark.VersionedFileSystem")
+      .set("fs.versioned.impl.disable.cache", "true")
```

Comment:
I don't think this should be set here as this isn't used in production code. Would be good to encapsulate these two steps in a method, e.g. `VersionedFileSystem.initialise()`. Then call it from `VersionedDataset.insert...` perhaps?